ui/src/app/api/entity.service.js

```diff
@@ -916,9 +962,7 @@ function EntityService($http, $q, $filter, $translate, $log, userService, device
                 entityFieldKeys.push(types.entityField.phone.keyName);
                 break;
             case types.entityType.entityView:
```

Comment:
Entity view has no label field, only name and type entity fields should be available